### ISSUE TITLE
[Issue]: Dependencies - runtime & build updates

### ISSUE THREAD

**Issue by kiritigowda** [ROCm/rpp#468]
## Missing the dependency details for RPP 

* C++ support
```
sudo apt install libstdc++-12-dev
```

* OpenMP support
```
sudo apt install libomp-dev
```

## RPP Test

Add test dependencies

### Operating System

ALL

### CPU

ALL

### GPU

AMD Instinct MI300

### ROCm Version

ROCm 6.0.0

### ROCm Component

rpp

**Comment by kiritigowda** [comment]
Dependency map - `readelf -d /opt/rocm/lib/librpp.so | grep NEEDED`
``` 
0x0000000000000001 (NEEDED)             Shared library: [libomp.so]
 0x0000000000000001 (NEEDED)             Shared library: [libamdhip64.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libstdc++.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libm.so.6]
 0x0000000000000001 (NEEDED)             Shared library: [libgcc_s.so.1]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
```

**Comment by kiritigowda** [comment]
@r-abishek @rrawther -- can we use AMD OMP(AOMP) for this project going forward?

**Comment by kiritigowda** [comment]
Part fix added with #472

**Comment by kiritigowda** [comment]
@LakshmiKumar23 with your issue #473 should be also combine AMD OMP as mentioned above?

**Comment by kiritigowda** [comment]
Fix added with #488
